### PR TITLE
Use reference_ops::Softmax() for RunSoftmaxFloatReference().

### PR DESCRIPTION
**Pull request by JerryShih**
If we use TFLITE_SOFTMAX_USE_UINT16_LUT table look up optimization
for softmax[1], the float look up table will not be initialized.
The optimized_ops::Softmax<float, float>() call will crash.
This commit turns to use reference_ops::Softmax<float, float>()
instead. That reference call should always be available.

[1]
https://github.com/tensorflow/tensorflow/blob/8f8b15c9cdd4f549354cc0e97c81ed9ab6096f67/tensorflow/lite/kernels/activations.cc#L602-L619

### CODE REVIEW

**Comment by abattery** [comment]
Is this test case failed at your end? Please elaborate more on describing the test failures including the error message and how to reproduce it.

**Comment by JerryShih** [comment]
> Is this test case failed at your end? Please elaborate more on describing the test failures including the error message and how to reproduce it.

@abattery
Sorry for the false alarm.
I have another overloading for the softmax kernel in my local branch. So, the test calls the wrong path and hits crash.
